inexor/network/doxygen-parser.conf.in

```diff
@@ -181,7 +181,7 @@ RECURSIVE              = YES
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories for example use the pattern */test/*
 
-EXCLUDE_PATTERNS       = */vendor/* */platform/*
+EXCLUDE_PATTERNS       = */vendor/* */platform/* */flex/* */node/*
```

Comment:
Neither /vendor nor /platform are existing anymore

Reply:
yes, I was thinking about this too.
i decided against removing it, since it does not do harm here and may save some trouble for current fucked up repos with left-over submodule folders. ("nested root")
Some later?

Reply:
changed